pom.xml

```diff
@@ -0,0 +1,405 @@
+        </developer>
+        <developer>
+            <name>Sam Hawker</name>
+            <email>sam.b.hawker@gmail.com</email>
+            <organization>IBM</organization>
+            <organizationUrl>https://www.ibm.com</organizationUrl>
+        </developer>
+        <developer>
+            <name>Jakub Stejskal</name>
+            <email>xstejs24@gmail.com</email>
+            <organization>Red Hat</organization>
+            <organizationUrl>https://www.redhat.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.version>3.8.1</maven.compiler.version>
+        <maven.surefire.version>3.0.0-M5</maven.surefire.version>
+        <maven.assembly.version>3.1.0</maven.assembly.version>
+        <maven.shade.version>3.1.0</maven.shade.version>
+        <maven.javadoc.version>3.1.0</maven.javadoc.version>
+        <maven.source.version>3.0.1</maven.source.version>
+        <maven.dependency.version>3.1.1</maven.dependency.version>
+        <maven.gpg.version>1.6</maven.gpg.version>
+        <maven.checkstyle.version>3.1.0</maven.checkstyle.version>
+        <spotbugs.version>4.0.3</spotbugs.version>
+        <sonatype.nexus.staging>1.6.3</sonatype.nexus.staging>
+        <jacoco.version>0.7.9</jacoco.version>
+        <license.maven.version>2.11</license.maven.version>
+        <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
+
+		<kafka.version>2.6.0</kafka.version>
+
+        <!-- property to skip surefire tests during failsafe execution -->
+        <skip.surefire.tests>${skipTests}</skip.surefire.tests>
+	</properties>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-mirror</artifactId>
+            <version>${kafka.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-log4j12</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-mirror-client</artifactId>
+            <version>${kafka.version}</version>
+        </dependency>
```

Comment:
Should these be configured as provided? We expect the resulting JAR to be copied to the Kafka `libs` directory where all of these dependencies will already be, or?

Reply:
You're right, I'll update the PR, I checked and updated the related PR in Strimzi repo.